Sofa/Component/AnimationLoop/src/sofa/component/animationloop/init.cpp

```diff
@@ -54,4 +55,10 @@ void init()
     }
 }
 
+const char* getModuleComponentList()
+{
+    /// string containing the names of the classes provided by the plugin
+    static std::string classes = core::ObjectFactory::getInstance()->listClassesFromTarget(sofa_tostring(MODULE_NAME));
```

Comment:
I dont think `sofa_tostring` is still required ?

Reply:
you're right